src/status_im2/subs/wallet_2/networks.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.subs.wallet-2.networks
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [quo2.foundations.resources :as resources]))
+
+(re-frame/reg-sub
+ :wallet-2/filtered-networks-by-mode
+ :<- [:wallet-2/networks]
+ (fn [[networks] [_ test?]]
```

Comment:
`test?` would eventually be pulled from a sub of its own. for now hardcoding this in as false

Reply:
turns out these values are the same so no hardcoding needed.